_posts/2017-02-09-CoC.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Our Code of Conduct
+---
+
+
+Dear MDAnalysis Community,
+
+MDAnalysis has adopted a
+[Code of Conduct]({{site.baseurl}}pages/conduct/) (CoC) to make
+explicit that we as a community value *all* our present and future
+members and embrace our diversity. We are committed to providing a
+productive, harassment-free environment for everyone
+
+Please read the full text at
+[http://mdanalysis.org/pages/conduct/]({{site.baseurl}}pages/conduct/).
+
+All core developers absolutely stand behind the CoC and are fully
+committed to ensuring and enforcing that *everyone* (founders,
+developers, anyone communicating on the lists or on GitHub, ...)
+respects each other as outlined in the CoC.
+
+But it's not only core developers who are responsible for making our's
+an inclusive community in which everyone feels they can productively
+participate. It actually need to be *all of us*. A community is based on
+the interactions of individuals and the sum of their values defines
+their community. The CoC means that everyone in MDAnalysis agrees to
+share a common set of core values and, importantly, to act according
+to these values in all interactions in our community.
+
+If you feel you suffered from harassment or intimidation or were made
+feel uncomfortable then please speak up: the CoC has links to
+[communicate safely]({{site.baseurl}}pages/conduct/#reporting) with a
+subset of the core developers who have taken on the responsibility to
+follow up on CoC violations. If you observe that someone else is being
+harassed please speak up – during the incident, if possible, and
+definitely through the above channels.
```

Comment:
In the CoC we have this sentence

> Not all problems need to be resolved via formal processes, and often a quick, friendly but clear word on an online forum or in person can help resolve a misunderstanding and de-escalate things.

I like it because it gives a self governance option as well as the formal process. I would like to keep that spirit and change the last sentence to


> If you observe that someone else is being harassed please speak up – during the incident, if possible, **or** through the above channels.